conventions-vclib/src/main/resources/vcLibVersions.properties

```diff
@@ -1,4 +1,4 @@
-signum=3.11.0
+signum=3.11.1
 supreme=0.6.0
```

Comment:
could also be 0.6.1, even though it does not matter in this project, as the transitive signum dependency is set pulled up anyways